vmm/src/vm.rs

```diff
@@ -968,6 +985,23 @@ impl Vm {
         Ok(EntryPoint { entry_addr })
     }
 
+    #[cfg(feature = "igvm")]
+    fn load_igvm(
+        igvm: File,
+        memory_manager: Arc<Mutex<MemoryManager>>,
+        cpu_manager: Arc<Mutex<cpu::CpuManager>>,
+    ) -> Result<EntryPoint> {
+        //TODO Generate the memory layout in arch/x86/layout.rs
+        // Related issue: https://github.com/cloud-hypervisor/cloud-hypervisor/issues/5993
+
```

Comment:
Please use "TODO: " to be consistent with the code base. Also, it is not clear that adding the layout to `arch/x86/layout.rs` would be the way to go. I'd simply say like: `TODO: see issue https://github.com/cloud-hypervisor/cloud-hypervisor/issues/5993`.